lib/dialects/abstract/query.js

```diff
@@ -23,6 +23,10 @@ class AbstractQuery {
       ...options
     };
     this.checkLoggingOption();
+
+    if (options.rawErrors) {
```

Comment:
I don't like changing methods at runtime but I get that it's easier to bypass method overrides this way.

For future us, can you document this line?

Reply:
Hi. I agree it is not the prettiest of things, but the alternative would be to check the flag in every single dialect. And newer dialects could easily miss that. So at least this solution is very simple.

I did see elsewhere in `lib/dialect/abstract/query-generator.js` the method `quoteIdentifier` being wrapped, in case you consider modifying methods is something you don't like or should not have,

What would you like to be documented there ?

Reply:
Simply explain what this line does (prevent descending classes from overriding this method) & why (so calling `.formatError` on the dialect returns the database error untranformed).